### PR TITLE
#827  Docs: Audited docs for `JSON.TYPE` command

### PR DESCRIPTION
**Pull request by onlybond**
closes issue #827 

Changes

- add Syntax section
- make Parameters section to table
- make Return Value section to table
- fix 127.0.0.1:6379 to 127.0.0.1:7379

### CODE REVIEW

**Review by apoorvyadav1111 — CHANGES_REQUESTED** [review]
Hi, Thank you for this change. I have few suggestions; DiceDB supports json natively and hence, we do not required external module such as DiceDBJSON. Would it be possible for you to remove any such mention in the document. Addtionally, please add a [Return values](https://dicedb.io/commands/set/#return-values),[Behaviour](https://dicedb.io/commands/set/#behaviour), and [Errors](https://dicedb.io/commands/set/#errors) section in this document.
This will ensure standardization amongst the commands.

**Comment by onlybond** [comment]
Thanks for the review @apoorvyadav1111 . i will get to it ASAP.

**Comment by lucifercr07** [comment]
@onlybond please address review comments from @apoorvyadav1111

**Comment by onlybond** [comment]
@lucifercr07 @apoorvyadav1111  the errors,  behavior and return values are already there in the document as i reviewed the documentation. i had made all the other changes which were mentioned in the issue #827

File: docs/src/content/docs/commands/JSON.TYPE.md [hunk 1]
```diff
@@ -66,44 +75,44 @@ If the path does not exist within the JSON document, the command will return `nu
...
-127.0.0.1:6379> JSON.TYPE user:1001 $.address.city
+127.0.0.1:7379> JSON.TYPE user:1001 $.address.city
 "string"
 ```
 
 ### Example 2: Non-Existent Path
 
 ```shell
-127.0.0.1:6379> JSON.TYPE user:1001 $.nonexistent
+127.0.0.1:7379> JSON.TYPE user:1001 $.nonexistent
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/JSON.TYPE.md:93]: Hi, for me, this is returning (empty array) in redis-cli. If I run the same set of commands in redis, its the same output. Can you please cross check and confirm on this.

File: docs/src/content/docs/commands/JSON.TYPE.md [hunk 2]
```diff
@@ -66,44 +75,44 @@ If the path does not exist within the JSON document, the command will return `nu
...
+127.0.0.1:7379> JSON.TYPE user:9999 $.name
 (nil)
 ```
 
 ### Example 4: Invalid JSONPath
 
 ```shell
-127.0.0.1:6379> JSON.TYPE user:1001 $..name
+127.0.0.1:7379> JSON.TYPE user:1001 $..name
 (error) ERR wrong number of arguments for 'json.type' command
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/JSON.TYPE.md:108]: I think there is some discrepancy in the heading for this example and the error message shown here. Please cross check once in the latest version of DiceDB.
Thanks

**Review by apoorvyadav1111 — CHANGES_REQUESTED** [review]
Hi, Thanks for the changes, I have left some comments for review. In addition to those, I have suggestion on changing the code block from shell to bash. Visually, I don't find any difference but since all other pages have used bash, I think we should maintain the same standard.

`If the path does not exist within the JSON document, the command will return `null`.`
Can you please check the validity of the above statement in the file. I could not leave a review  as this was not part of the change.

**Comment by apoorvyadav1111** [comment]
Hi @onlybond , Hope you are well. We are excited to know more about your progress on this issue. As you know, we would like to close this issue soon and if you need any assistance, don't hesitate to get in touch with me or anyone else on the discord community. Let's close this one together soon.

Thanks

**Comment by onlybond** [comment]
hi @apoorvyadav1111 actually i am unable to get a server running so cannot find the exact errors which you told about

**Comment by apoorvyadav1111** [comment]
Hi @onlybond , Please reach out to me on discord, we should be able to get the issues sorted and close your PR soon.

File: docs/src/content/docs/commands/JSON.TYPE.md (hunk 1 above)
Comment (onlybond) [reply docs/src/content/docs/commands/JSON.TYPE.md:93]: yes i have updated the output in my latest commit

File: docs/src/content/docs/commands/JSON.TYPE.md [hunk 3]
```diff
@@ -65,45 +74,45 @@ If the path does not exist within the JSON document, the command will return `nu
...
 ```
 
 ### Example 2: Non-Existent Path
 
-```shell
-127.0.0.1:6379> JSON.TYPE user:1001 $.nonexistent
-(null)
+```bash
+127.0.0.1:7379> JSON.TYPE user:1001 $.nonexistent
+(empty list or set)
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/JSON.TYPE.md:94]:
```suggestion
(empty array)
```

**Review by apoorvyadav1111 — APPROVED** [review]
Hi @onlybond, Thanks for making the changes. I have edited one line to ensure consistency with other docs. I hope that's okay. I am approving and merging this PR.

**Pull request merged**